src/core/lib/iomgr/endpoint.h

```diff
@@ -47,6 +47,7 @@ struct grpc_endpoint_vtable {
   grpc_resource_user* (*get_resource_user)(grpc_endpoint* ep);
   char* (*get_peer)(grpc_endpoint* ep);
   int (*get_fd)(grpc_endpoint* ep);
+  bool (*can_track_err)(grpc_endpoint* ep);
```

Comment:
Is there any actual business logic needed here? This can just be a bool, no need for a fn ptr

Reply:
Oops nevermind, read a bit further and saw the actual function body